checkIn/checkIn.py

```diff
@@ -219,6 +221,21 @@ class CardScan(Base):
 	def __repr__(self):
 		return "<CardScan %d at %s>" % (self.card, self.time)
 
+class Warning(Base):
```

Comment:
Warning count should be visible on the user lookup page and maybe other places as well, maybe the user search page too

Reply:
I added it to the search page, where do you think I should put it on the lookup page?

Reply:
Maybe just like a little badge on the side of the row that shows the number of warnings the person's received. Just so someone can see it at a glance whether or not someone routinely causes issues.

Reply:
I think that's what I was calling the search page?  Why don't you check the latest commit and see if it does what you wanted